guide/writing-code/instructions/mov.md

```diff
@@ -65,8 +71,15 @@ mov rbx, 5;
 
 mov rax, rbx;
 ```
+
+<details>
+<summary><i>At the end of this example, what do you think the value of rax will be?</i></summary>
+
+<br />
 _Even though we set `rax` to `3` in the beginning, at the end of this example, `rax` is `5` since we set `rax` to the contents of `rbx`._
```

Comment:
```suggestion
<i>Even though we set `rax` to `3` in the beginning, at the end of this example, `rax` is `5` since we set `rax` to the contents of `rbx`.</i>
```